src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -76,53 +76,50 @@ function toggleChevronImage(panelCollapse, toggleChevron) {
  * Expands panel, showing the spreadsheet interface.
  */
 function expandStudentsPanel(panelCollapse) {
-    $(panelCollapse[0]).collapse('show');
-    $(panelCollapse[0]).addClass('checked');
+    panelCollapse.collapse('show');
 }
 
 /**
  * Collapses panel, hiding the spreadsheet interface.
  */
 function collapseStudentsPanel(panelCollapse) {
-    $(panelCollapse[0]).collapse('hide');
-    $(panelCollapse[0]).removeClass('checked');
+    panelCollapse.collapse('hide');
 }
 
 /**
  * Expands/Collapses the panel depending on the current state of the panel.
  */
 function toggleStudentsPanel($panelHeading, panelCollapse, displayIcon, toggleChevron) {
+    let isPanelExpanded = false;
     displayIcon.html('');
-    if ($(panelCollapse[0]).hasClass('checked')) { // panel is shown
+    if (panelCollapse.hasClass('in')) { // panel is shown
         collapseStudentsPanel(panelCollapse);
     } else {
         expandStudentsPanel(panelCollapse);
+        isPanelExpanded = true;
```

Comment:
A better pattern to use is
```
const isPanelExpanded = panelCollapse.hasClass('in');
if (isPanelExpanded) ...
```
Why? Because this way, the variable's value never changes. One hard thing about programming, reading code, finding bugs etc. is in tracking the states of variables. If your variable changes state a lot, it is harder to track.

Also, now that `collapseStudentsPanel`, `expandStudentsPanel` as well as `toggleChevronImage` is so simple, do you think they should just be removed and their content added here instead? We wouldn't need the variable `isPanelExpanded` as well.